util/argo/argo.go

```diff
@@ -146,12 +146,13 @@ func getCAPath(repoURL string) string {
 	return ""
 }
 
+// TODO: Figure out the correct way to distinguish between SSH and HTTPS repos
```

Comment:
TODO?

Reply:
Yes, this is something that works for now, but doesn't feel very correct. Need to think about this a little more.